demo_classification_opvn.py

```diff
@@ -0,0 +1,32 @@
+import numpy as np
+import cv2 as cv
+import argparse
+import os
```

Comment:
There is no much difference between `demo_classification_tf` and `demo_classification_opvn` - please create a single `demo.py` which accepts flag for target engine